prefetch.bs

```diff
@@ -225,19 +228,40 @@ A <dfn export>cross-origin prefetch IP anonymization policy</dfn> has an <dfn ex
   1. Return false.
 </div>
 
+A <dfn export>prefetch controller</dfn> is a [=struct=] which allows managing a prefetch operation. It has the following [=items=]:
+* <dfn for="prefetch controller">state</dfn> (default `"ongoing"`), which is `"ongoing"` or `"aborted"`
+
+  <div class="note">The state is `"ongoing"` even after prefetch is complete, as long as it was not aborted by the author.</div>
+* <dfn for="prefetch controller">fetch controller</dfn> (default null), which is a [=fetch controller=] or null
+* <dfn for="prefetch controller">prefetch record</dfn> (default null), which is a [=prefetch record=] or nulll
+* <dfn export for="prefetch controller">document</dfn>, which is a {{Document}}
+
+  <div class="note">This is the {{Document}} whose [=prefetch buffer=] contains this's [=prefetch controller/prefetch record=], if any.
```

Comment:
```suggestion
  <div class="note">This is the {{Document}} whose [=prefetch buffer=] contains the [=prefetch controller/prefetch record=], if any.
```

("this" seems too JavaScript/Web IDL-ey to me)

Reply:
Obsoleted by refactor.